server/src/models/Review.js

```diff
@@ -0,0 +1,63 @@
+const { BelongsToOneRelation } = require('./Model')
+const Model = require('./Model')
+
+class Review extends Model {
+  static get tableName() {
+    return 'reviews'
+  }
+
+  static get jsonSchema(){
+    return {
+      type: 'object',
+      required: ['rating', 'userId', 'productId'],
+      properties: {
+        rating: {
+          type: ['integer', 'string'],
+          minimum: 1,
+          maximum: 5
+        },
+        title: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 255
+        },
+        content: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 1000
+        },
+        userId: {
+          type: ['integer', 'string']
+        },
+        productId: {
+          type: ['integer', 'string']
+        }
```

Comment:
while this would be ideal, and I'll allow it, know that these last two validations will prevent you from running the following, and may need to be removed if you want this functionality:

recipe.$relatedQuery("product").insert({ .... }) 

you choice though!